demo/full/scripts/controllers/PlayerKnobsInfos.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import withModulesState from "../lib/withModulesState.jsx";
+// import AudioBitrateKnob from "./knobs/AudioBitrate.jsx";
+// import VideoBitrateKnob from "./knobs/VideoBitrate.jsx";
+// import LanguageKnob from "./knobs/AudioTrack.jsx";
+// import SubtitlesKnob from "./knobs/Subtitles.jsx";
+// import VideoTrack from "./knobs/VideoTrack.jsx";
+// import PlaybackRateKnob from "./knobs/SpeedKnob.jsx";
+
+const PlayerKnobsInfos = ({
+  shouldDisplay,
+  close,
+  // player,
+  // availableVideoTracks,
+  liveGap,
+  isContentLoaded,
+}) => {
+
+  if (!isContentLoaded) {
+    return null;
+  }
+
+  const className = "player-knobs" + (shouldDisplay ? " fade-in-out" : "");
```

Comment:
Wouldn't that be more clear to render it or not in its parent?
Or is there an issue then for the "fade-in-out" effect?